templates/master/routes/examples/list.vm

```diff
@@ -1,4 +1,4 @@
-#set ( $root="https://${!context.apiId}.execute-api.${AWS::Region}.amazonaws.com/${!context.stage}")
+#set ($root="https://${!context.domainName}/${!context.stage}")
```

Comment:
How this world work for the default API url (`$root="https://${!context.apiId}.execute-api.${AWS::Region}.amazonaws.com/${!context.stage}"`)?, where did you set `context.domainName` to have the correct value?